Source/monster.cpp

```diff
@@ -1803,50 +1802,53 @@ void GroupUnity(Monster &monster)
 	}
 }
 
-bool RandomWalk(int monsterId, Direction md)
+/**
+ * @brief Walks the monster in the direction md.
+ *
+ * Take the following monster for example:
+ *
+ * @code{.unparsed}
+ * ↖️ ⬆️ ↗️
+ * ⬅️ 😈 ➡️
+ * ↙️ ⬇️ ↘️
+ * @endcode

```

Comment:
GitHub is not rendering the emojis correctly for me, but it looks correct in both my editor (CLion) and in Doxygen:

![image](https://user-images.githubusercontent.com/288227/178475998-a40f3f06-b7aa-4398-a8da-6c5c84ab8c7c.png)